testing/src/main/java/net/consensys/linea/testing/ExecutionEnvironment.java

```diff
@@ -67,8 +68,9 @@ public static void checkTracer(
       logger.ifPresent(log -> log.debug("trace written to {}", finalTraceFilePath));
       CorsetValidator.Result corsetValidationResult = corsetValidator.validate(traceFilePath);
       assertThat(corsetValidationResult.isValid())
-          .withFailMessage("Corset validation result: %s", corsetValidationResult.toString())
+          .withFailMessage(CORSET_VALIDATION_RESULT + "%s", corsetValidationResult.corsetOutput())
           .isTrue();
+      return corsetValidationResult.corsetOutput();
```

Comment:
This will only be returned when the above assertion is true. When it is true returning this output adds no value.